microshift_getting_started/microshift-architecture.adoc

```diff
@@ -2,30 +2,63 @@
 [id="microshift-architecture"]
 = Architecture
 include::_attributes/attributes-microshift.adoc[]
+include::_attributes/common-attributes.adoc[]
 :context: microshift-architecture
 
 toc::[]
 
-Learn the specifics of {product-title} architecture including design intent, how it differs from {OCP}, and API compatibility.
+Learn the specifics of {product-title} architecture including design intent, how it differs from {oke}, and API compatibility.
 
 [id="microshift-architectural-design_{context}"]
 == Architectural design
-{product-title} is a single-node container orchestration runtime designed to extend the benefits of using containers for running applications to low-resource edge environments. {product-title} contains only the APIs and features essential to operating in edge and small form factor computing environments. The following operational differences from {OCP} can help you understand where {product-title} can be deployed:
+{product-title} is a single-node container orchestration runtime designed to extend the benefits of using containers for running applications to low-resource edge environments. Because {product-title} is primarily as a platform for deploying applications, only the APIs and features essential to operating in edge and small form factor computing environments are included.
```

Comment:
```suggestion
{product-title} is a single-node container orchestration runtime designed to extend the benefits of using containers for running applications to low-resource edge environments. Because {product-title} is primarily a platform for deploying applications, only the APIs and features essential to operating in edge and small form factor computing environments are included.
```